frontend/src/metabase/nav/components/NewItemButton/NewItemButton.tsx

```diff
@@ -12,16 +10,13 @@ export interface NewItemButtonProps {
 }
 
 const NewItemButton = ({ collectionId }: NewItemButtonProps) => {
-  const dispatch = useDispatch();
-
   return (
     <NewItemMenu
       trigger={
         <NewButton primary icon="add" aria-label={t`New`}>
           <NewButtonText>{t`New`}</NewButtonText>
         </NewButton>
       }
-      appendMenuItems={PLUGIN_GO_MENU.getMenuItems(dispatch)}
```

Comment:
Has no effect because `appendMenuItems` is missing from the component implementation.